test/test_idb_companion.js

```diff
@@ -0,0 +1,110 @@
+var idbCompanion = require('../lib/idb_companion.js');
+var sinon = require('sinon');
+var assert = require('assert');
+var process = require('process');
+var childProcess = require('child_process');
+
+
+afterEach(() => {
```

Comment:
For another MR, let's add a GitHub Actions workflow that runs this on every commit.